eahub/templates/eahub/profile.html

```diff
@@ -22,6 +22,15 @@
     </div>
     <div class="col-xs-12 col-md-6">
       <h1 class="word-break">{{ profile.name }}</h1>
+      {% if profile.linkedin_url %}
```

Comment:
As you said on slack, positioning it here is not ideal. 
Maybe we could place it in the summary box? Or we add an extra box under the summary box for URLs?

Reply:
My first preference is if there was another panel below Summary called Links. Both of these icons could be placed there. If this was troublesome for any reason, the current position could work if there was a <br> to visually push the other row of buttons underneath. We should also use this color for the icons

![image](https://user-images.githubusercontent.com/14195560/63249323-2a5f0f80-c261-11e9-96f4-083eb1b294fb.png)

Because we don't currently offer messaging (and "soon" isn't this year anymore), we could also use the purple buttons to lead to social platforms the links to which people could upload in their profiles.
 
![image](https://user-images.githubusercontent.com/14195560/63249784-1071fc80-c262-11e9-9025-e9f0d1f82ef2.png)

So that when I'm visiting my own profile, the buttons say i.e. "Update LinkedIn URL", "Update Facebook URL". When I'm visiting someone else's profile, they say "LinkedIn", "Facebook", etc.